app/server-utils/stampy.ts

```diff
@@ -153,7 +189,38 @@ export const loadOnSiteAnswers = withCache('onSiteAnswers', async () => {
 
 export const loadAllQuestions = withCache('allQuestions', async () => {
   const rows = await getCodaRows(ALL_ANSWERS_TABLE)
-  return rows.map(({name}) => name)
+  return rows.map((r) => convertToQuestion(r.name, r.values))
+})
+
+const extractMainQuestion = (question: string | Entity | null): string | null => {
+  switch (typeof question) {
+    case 'string':
+      return extractText(question)
+    case 'object':
+      return question?.name || null
+    default:
+      return question
+  }
+}
+const toTag = (r: CodaRow, nameToId: Record<string, string>): Tag => ({
+  rowId: r.id,
+  tagId: r.values['Tag ID'],
+  name: r.name,
+  url: r.browserLink,
+  internal: r.values['Internal?'],
+  questions: r.values['Questions tagged with this']
+    .map(({name}) => ({title: name, pageid: nameToId[name]}))
+    .filter((q) => q.pageid),
+  mainQuestion: extractMainQuestion(r.values['Main question']),
+})
+export const loadAllTags = withCache('allTags', async () => {
+  const rows = await getCodaRows(TAGS_TABLE)
+  const questions = await loadAllQuestions({url: ''})
```

Comment:
this `{url: ''}` is to use the cacheing mechanism for all the questions. I don't like the added complexity, but it makes things a bit easier